install

```diff
@@ -101,61 +101,10 @@ X-GNOME-Autostart-enabled=true
 Hidden=false
 NoDisplay=false" > ~/.config/autostart/pi-apps-updater.desktop
 
-mkdir -p "${DIRECTORY}/data" && cd "${DIRECTORY}/data" || error "Failed to make and enter ${DIRECTORY}/data directory!"
-mkdir -p installed-packages preload settings status update-status categories
-cd $HOME
-
-#hide template app by default
-"${DIRECTORY}/etc/categoryedit" "template" 'hidden' &>/dev/null
-
-#hide duplicates if running in twisteros
-if [ -f /usr/local/bin/twistver ] && [[ $(twistver) == "Twister OS version"* ]]; then
```

Comment:
what is your plan with removing this from the installer?
have you given into my category overrides PR? or have you just given up on the twister duplicates

Reply:
@Botspot I would like to have this merged soon. can you answer the above so I know what changes need to be made (if any)?